app/index.html

```diff
@@ -13,7 +14,27 @@ <h1>HELLO WORLD!</h1><i class="fa fa-hand-spock-o" aria-hidden="true"></i>
   	<div class="header header__second"></div>
   	<div class="header header__third"></div>
   	<div class="header header__fourth"></div>
-  	<div class="header header__fifth"></div>
+  	<div class="header header__fifth">
+			<main class="fifth">
```

Comment:
it think it is unnecessary to make such big indents in nesting. We can discuss indents on the next lesson

Reply:
I don't have such big indents in my computer - this's one tab.

Reply:
you can always set indents in your editor. You can leave it for now as it is and I think we should discuss this with all team on the lesson

Reply:
Okey, it'll be useful.